buildall.sh

```diff
@@ -16,7 +16,12 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 PIDS=()
-
+# some systems do not have enough tty's available to run all the
+# dumbinit tests in parallel. If the tty tests are failing try
+# setting SERIAL_BUILD=1
+if [ "$SERIAL_BUILD" == "" ]; then
+    SERIAL_BUILD=0
```

Comment:
Some of the lines you added are spaces, some are tabs. But other than that we should at least have consistent indentation throughout the file.